test/e2e/config/ironic.yaml

```diff
@@ -22,29 +22,17 @@ variables:
   DEPLOY_CERT_MANAGER: "true"
   BMO_KUSTOMIZATION: "../../config/overlays/e2e"
   IRONIC_KUSTOMIZATION: "../../ironic-deployment/overlays/e2e"
+
   # These settings are for the separate cluster that are used for upgrade tests
   UPGRADE_DEPLOY_IRONIC: "true"
   UPGRADE_DEPLOY_BMO: "true"
   UPGRADE_DEPLOY_CERT_MANAGER: "true"
   UPGRADE_BMO_KUSTOMIZATION_FROM: "../../config/overlays/e2e-release-0.4"
 
-  # Test credentials. The tests will create a BMH with these.
-  BMC_USER: admin
-  BMC_PASSWORD: password
-  # During development, select the preferred BMC interface when running tests from the command line:
-  # Note: These settings are only applicable for command line executions, not when running via the ci-e2e.sh script.
-  # Uncomment the desired BMC_ADDRESS line:
-  # BMC_ADDRESS: "ipmi://192.168.222.1:16230"
-  BMC_ADDRESS: "redfish+http://192.168.222.1:8000/redfish/v1/Systems/bmo-e2e-0"
-  BOOT_MAC_ADDRESS: "00:60:2f:31:81:01"
-  EXPECTED_HOST_NAME: "bmo-e2e-0"
-
   IMAGE_URL: "http://192.168.222.1/cirros-0.6.2-x86_64-disk.img"
   IMAGE_CHECKSUM: "c8fc807773e5354afe61636071771906"
   CERT_MANAGER_VERSION: "v1.13.1"
   SSH_CHECK_PROVISIONED: "true"
-  IP_BMO_E2E_0: "192.168.222.122"
-  SSH_PORT: "22"
```

Comment:
Do all tests pass without `SSH_PORT` ? It is used when checking to verify the node's boot source. Maybe it used 22 by default. But I think it would be good to still have it because the variable is used in the go code and someone might want to use another port sometime?

Reply:
Oh NVM! I see that you have changed the code.